model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/schema.proto

```diff
@@ -31,6 +31,8 @@ option go_package = "github.com/apache/beam/sdks/v2/go/pkg/beam/model/pipeline_v
 option java_package = "org.apache.beam.model.pipeline.v1";
 option java_outer_classname = "SchemaApi";
 
+import "org/apache/beam/model/pipeline/v1/beam_runner_api.proto";
```

Comment:
Do we need this dependency ?

Reply:
Yes, we need it for using bean_urn extension.